README.md

```diff
@@ -55,15 +56,15 @@ NOTE: You will likely need to change the *service_name* from *dpaste-<username>*
 
 Also, if you skipped the build step above you can use the pre-built AMIs to deploy with:
 
- * **us-east-1**: ami-ebcab6fc
- * **us-west-2**: ami-eea67b8e
+ * **us-east-1**: ami-201c5337
+ * **us-west-2**: ami-cfc51daf
 
 #### Terraform Get
 
 Terraform makes heavy uses of modules, and before we can successfully deploy an application with it, we need it to first download the modules it needs
 
 ```bash
-$> terraform get -update=true
+$> terraform get --update=true
```

Comment:
Terraform generally prefers the '-' single dash for its options. Why the change? I didn't even know it was supported.

In any case, TF's help always presents options with a single dash, so I'd stick with that.